### ISSUE TITLE
Add possibility to only use FQDN without SSH_SERVER_PORT env

### ISSUE THREAD

**Issue by Riushda** [Ravinou/borgwarehouse#237]
When SSH_SERVER_PORT env is not set, it default to 22 [here](https://github.com/Ravinou/borgwarehouse/blob/main/pages/api/account/getWizardEnv.js#L21) and is still used in the web command hints. In my local setup, I use  a reverse proxy with domain name which forward automatically to correct ip and port. Thus it would be very handy to have to possibility to only specify FQDN  url without Server port. 

I'm not sure exactly what could be the best solution, either do not use port if not set (instead of using port 22 by default) or introducing a third env variable that is supposed to already contain the hostname and port (and use this one if set). I think the latter would ensure backward compatibility with existing setup.

**Comment by Ravinou** [comment]
Your case is interesting and deserves to be treated. I'll think about the best way to do it and incorporate the changes. Stay tuned ;)

**Comment by Ravinou** [comment]
Hi @Riushda ! It will work with this in your `.env`

```
SSH_SERVER_PORT=false
SSH_SERVER_PORT_LAN=false
```

It will be on the **next release** 🚀

**Comment by Ravinou** [comment]
I've improved the code for greater clarity. To hide the SSH port in quickcommands and wizard, simply add the environment variable : 

```
# Hide the SSH port in the UI : quickcommands & wizard
NEXT_PUBLIC_HIDE_SSH_PORT=true
```


This makes things clearer, and better compatibility with existing installations.

It will be available in v2.4.0

**Comment by Ravinou** [comment]
[Published in latest release.](https://github.com/Ravinou/borgwarehouse/releases/tag/v2.4.0)
